app/controllers/patients_controller.rb

```diff
@@ -497,7 +497,7 @@ def bulk_update
       patients.each do |patient|
         # We never want to update closed records monitoring status via the bulk_update
         update_params = patient.monitoring ? params : closed_params
-        update_monitoring_fields(patient, update_params, non_dependent_patient_ids.include?(patient[:id]) ? :patient : :dependent,
+        update_monitoring_fields(patient, update_params, non_dependent_patient_ids.include?(patient[:id]) ? patient.id : nil,
```

Comment:
On this line do you ever want to pass `nil` for the `household`? Isn't that field always meant to now be the `id` of the HOH? I'm noticing that since this is `nil` here, the checks in `history.rb` for:
```ruby
history[:household_status] == history[:patient].responder_id
```
end up failing, because `responder_id` is an id, and `household_status` is `nil`. This then causes a history message on a dependent to not have any note about where that change came from:
![image](https://user-images.githubusercontent.com/52747882/124811789-d9c62d80-df30-11eb-9f19-b30281452428.png)